herbert_core/classes/MPIFramework/@MessagesFilebased/MessagesFilebased.m

```diff
@@ -273,14 +273,22 @@ function set_is_tested(obj,is_tested)
             %
```

Comment:
it would be helpful to include something in the function comment describing what format this generated folder name takes; the code below assembles a number of different elements into the name with various code paths through it.

Reply:
hopefully added